src/assets/cli-usage.txt

```diff
@@ -0,0 +1,35 @@
+Nitric - The fastest way to build serverless apps
+
+To start with nitric, run the 'nitric new' command:
+
+    $ nitric new
+
+This will guide you through project creation, including selecting from available templates.
+
+Common commands in the CLI that you’ll be using:
+
+
```

Comment:
Should this be listing the common commands? Might be problem with CLI not this PR.

Reply:
fixed by new release of cli